tasks/package.yaml

```diff
@@ -9,9 +9,9 @@
     - sudo_pkg
 
 - name: Install sudo for RedHat
-  yum:
-    name: sudo
-    state: present
+  action:
+    "{{ ansible_pkg_mgr }} pkg=sudo
+    state=present"
```

Comment:
Btw. the quotation here is wrong. And using one-line Ansible params (`state=present` instead of `state: present`) is bad practice.